src/pageObject.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
```

Comment:
The 'ArticlePage' constructor should call `super(url)` and then append the 'slug' to the 'url' property of the instance. This will ensure that the 'url' property of the 'PageObject' class is correctly set to the base URL.

Reply:
it`s super(idea). Done